wercker.yml

```diff
@@ -46,7 +46,10 @@ build:
           CS_POM_VERSION=$(mvn -q -Dexec.executable='echo' -Dexec.args='${project.version}' --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
           echo CS_version: ${CS_POM_VERSION}
           for i in 1 2 3 4 5; do git clone https://github.com/pgjdbc/pgjdbc.git && break || sleep 15; done
-          cd pgjdbc/pgjdbc
+          cd pgjdbc/pgjdbc/src/main/checkstyle
```

Comment:
please avoid change of this line `cd ....`, HACK lines(wget+patch) should not affect other lines to be easily removed.